sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -213,9 +214,43 @@ static rom_error_t mask_rom_boot(const manifest_t *manifest,
   sec_mmio_check_values(rnd_uint32());
   sec_mmio_check_counters(/*expected_check_count=*/3);
 
+  /* These symbols are defined in
+  * `opentitan/sw/device/silicon_creator/mask_rom/mask_rom.ld`, and describes the
+  * location of the flash header.
+  */
+  extern uint32_t _rom_ext_virtual_address;
+  extern uint32_t _rom_ext_virtual_size;
+
+  #define  GET_VMA_FROM_LMA(lma_addr_) \
```

Comment:
I think we should make this an inline function rather than a macro but otherwise lgtm.